quickwit/quickwit-metastore/src/tests/mod.rs

```diff
@@ -105,13 +102,13 @@ async fn create_channel(client: tokio::io::DuplexStream) -> anyhow::Result<Chann
     Ok(channel)
 }
 
-crate::metastore_test_suite!(
-    quickwit_proto::metastore::MetastoreServiceGrpcClientAdapter<
-        quickwit_proto::metastore::metastore_service_grpc_client::MetastoreServiceGrpcClient<
-            quickwit_proto::tonic::transport::Channel,
-        >,
-    >
-);
+// crate::metastore_test_suite!(
+//     quickwit_proto::metastore::MetastoreServiceGrpcClientAdapter<
```

Comment:
can we reenable this?

Reply:
I will in my next PR. It's tricky because we only have access to the client here but the tests required the insertion of shards in the server via an API that does not exist client-side and makes no sense to add.